### PR TITLE
Prepare for DistributedActor integration with Swift 5.7

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -238,22 +199,18 @@ var dependencies: [Package.Dependency] = [
...
-dependencies += [
-    .package(url: "https://github.com/apple/swift-argument-parser", .upToNextMinor(from: "0.3.2")), // not API stable, Apache v2
+    .package(url: "https://github.com/apple/swift-distributed-tracing.git", from: "0.3.0"),
 ]
 
 // swift-syntax is Swift version dependent, and added as such below
 #if swift(>=5.6)
 dependencies.append(
-    // Works with: swift-PR-39654-1170.xctoolchain
+      // Works with: swift-PR-39654-1170.xctoolchain
```
Comment (tomerd) [inline Package.swift:213]: is swift syntax still needed? since no code generator

Comment (ktoso) [reply Package.swift:213]: Right, no need for it -- thanks for spotting :)

**Comment by ktoso** [comment]
Okey so everything works fine with

```
-> % export TOOLCHAIN=/Library/Developer/Toolchains/swift-PR-42601-146.xctoolchain; $TOOLCHAIN/usr/bin/swift build --build-tests
Building for debugging...
Build complete! (0.22s)
```

So it's just a matter of cleanup and going through all tests and internals now... 

I think nightly is new enough already... let's check

**Comment by ktoso** [comment]
@swift-server-bot test this please

**Comment by ktoso** [comment]
@swift-server-bot test this please

**Pull request merged**